idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+
+        """
+        self.liquid_phase = ControlVolume1DBlock(default={
+            "transformation_method": self.config.transformation_method,
+            "transformation_scheme":
+                self.config.liquid_side.transformation_scheme,
+            "finite_elements": self.config.finite_elements,
+            "collocation_points": self.config.collocation_points,
+            "dynamic": self.config.dynamic,
+            "has_holdup": self.config.has_holdup,
+            "area_definition": DistributedVars.variant,
+            "property_package": self.config.liquid_side.property_package,
+            "property_package_args":
+                self.config.liquid_side.property_package_args})
+
+        self.liquid_phase.add_geometry(flow_direction=set_direction_liquid,
+                                       length_domain_set=self.config.
+                                       length_domain_set)
+
+        self.liquid_phase.add_state_blocks(
+            information_flow=set_direction_liquid,
+            has_phase_equilibrium=False)
+
+        self.liquid_phase.add_material_balances(
+            balance_type=MaterialBalanceType.componentTotal,
+            has_phase_equilibrium=False,
+            has_mass_transfer=True)
+
+        self.liquid_phase.add_energy_balances(
+            balance_type=EnergyBalanceType.enthalpyTotal,
+            has_heat_transfer=True)
+
+        self.liquid_phase.apply_transformation()
+
+        # Add Ports for vapor side
+        self.add_inlet_port(name="vapor_inlet", block=self.vapor_phase)
+        self.add_outlet_port(name="vapor_outlet", block=self.vapor_phase)
+
+        # Add Ports for liquid side
+        self.add_inlet_port(name="liquid_inlet", block=self.liquid_phase)
+        self.add_outlet_port(name="liquid_outlet", block=self.liquid_phase)
+
+    # ==========================================================================
+        """ Add performace equation method"""
+        self._make_performance()
+
+    def _make_performance(self):
+        """
+        Constraints for unit model.
+
+        Args: None
+
+        Returns: None
+
+        """
+
+        # ======================================================================
+        # Custom Sets
+        vap_comp = self.config.vapor_side.property_package.component_list
+        liq_comp = self.config.liquid_side.property_package.component_list
+        equilibrium_comp = vap_comp & liq_comp
+        solvent_comp_list = \
+            self.config.liquid_side.property_package.solvent_set
+        solute_comp_list = self.config.liquid_side.property_package.solute_set
+        liq_comp_apparent = self.config.liquid_side.property_package._apparent_set
+        vapor_phase_list_ref = \
+            self.config.vapor_side.property_package.phase_list
+        liquid_phase_list_ref = \
+            self.config.liquid_side.property_package.phase_list
+
+        # Packing  parameters
+        self.eps_ref = Param(initialize=0.97,units=None,
```

Comment:
Units should be `pyunits.dimensionless` to be explicit (i.e. it is unitless, versus we were just too lazy to specify).

I would also suggest `packing_voidage` instead of `eps_ref`, as we use `eps` for a lot of purposes.